components/renku_data_services/data_connectors/db.py

```diff
@@ -263,6 +314,9 @@ async def update_data_connector(
             raise errors.MissingResourceError(message=not_found_msg)
         old_data_connector = data_connector.dump()
 
+        if old_data_connector.project and old_data_connector.project.slug != patch.project_slug:
+            raise NotImplementedError("Moving the data connector to another project or namespace is not supported")
```

Comment:
```suggestion
            raise NotImplementedError("Moving a data connector to another project or namespace is not supported")
```

otherwise it sounds like there's something wrong with this dataconnector specifically that makes it so it can't be moved